### PR TITLE
Restyle homepage

### PR DESCRIPTION
**Pull request by Infinical**
#### Issue addressed

fixes #

### CODE REVIEW

**Comment by mattwr18** [comment]
Could you please upload some screen shots @Infinical

**Comment by Infinical** [comment]
Screenshot at 2019-04-09 17-05-55
Screenshot at 2019-04-09 17-05-37
Screenshot at 2019-04-09 17-05-13

**Comment by mattwr18** [comment]
great! thanks @Infinical 
will you be able join the #anvil meeting today?

Going back to what you were asking, I think it will definitely be better to separate the Modals, if possible.
Then as you said we can maybe style them similarly to how they are in production or maybe even a better design?

Also, it would be great if you could find a way to add some padding on the left of the images, and maybe change the transparency of the background image?

File: src/components/homepage/HomepageModal.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
+$b-r: 5px;
+
+
+
```
Comment (joaopapereira) [inline src/components/homepage/HomepageModal.scss:4]:
```suggestion
```

File: src/components/homepage/Homepage.css [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .landing-page-background {
   margin: 0;
   padding: 0;
-  background-image: url("../../images/AgileJourneyBackground.svg");
+  background-image: url("../../images/landing-page-background.png") !important;
```
Comment (joaopapereira) [inline src/components/homepage/Homepage.css:4]: Do we need the important?

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Please review my comments

File: src/components/homepage/HomepageModal.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+.card {
+  display: flex;
+  flex: 0 0 auto;
+  background: white;
+  opacity: 30%;
```
Comment (mattwr18) [inline src/components/homepage/HomepageModal.scss:9]: it looks like we can add `padding-left` here

**Review by mattwr18 — CHANGES_REQUESTED** [review]

**Comment by mattwr18** [comment]
@Infinical I'm gonna close this as I put in a separate `PR` that was merged in building off of yours and @Kachulio1's work

**Pull request closed without merging**